tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -464,6 +470,13 @@ func (az *AzureProvider) constructStorageProfile(imageId string) (storageProfile
 	return storageProfile
 }
 
+// getWorkerBackendPoolID gets the backend pool id of the worker node loadbalancer
+func (az *AzureProvider) getWorkerBackendPoolID() *string {
+	var wbpi = fmt.Sprintf("/subscriptions/%s/resourceGroups/%[2]s-rg/providers/"+
+		"Microsoft.Network/loadBalancers/%[2]s/backendAddressPools/%[2]s", az.subscriptionID, az.infraID)
```

Comment:
I did not pick up on this when you showed us code yesterday. Are you assuming that index 2 will always be the worker LB? You cannot make that assumption. You need to make a call to get all the LBs in the RG and then pick the one being used by the Linux worker nodes.